docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc

```diff
@@ -104,6 +104,21 @@ where included fields contain an array are also ignored. Documents in the
 destination index that don't contain a results field are not included in the 
 {classanalysis}.
 
+[float]
+[[dfa-classification-imbalanced-classes]]
+=== Imbalanced class sizes affect {classification} performance
+
+If your training data is very imbalanced, then {classanalysis} may not provide 
+good predictions. The system tries to mitigate the effects of imbalanced 
+training data by maximizing the minimum recall of any class. It means that it 
+tries to get correct the highest number of examples in the minority classes. 
```

Comment:
I think I'd modify these sentences slightly. Here "the system" is really training so I'd probably write something like:
"Training tries to mitigate the effects of imbalanced training data by maximizing the minimum recall of any class. For imbalanced training data, this means that it will try to balance the proportion of values it correctly labels in the minority and majority class."